p2p/errors.go

```diff
@@ -99,3 +99,9 @@ var ErrWatchdogAlreadyStarted = errors.New("peer discovery watchdog is already s
 
 // ErrWatchdogNotStarted signals that a peer discovery watchdog is not started
 var ErrWatchdogNotStarted = errors.New("peer discovery watchdog is not started")
+
+// ErrContextDone signals that an opertion was cnceled by context done
+var ErrContextDone = errors.New("context done")
+
+// ErrTimeout signals a timeout
+var ErrTimeout = errors.New("timeout")
```

Comment:
maybe you can add more details in this err, if you get this error to know from where it comes

Reply:
I want to keep it generic